index.qmd

```diff
@@ -106,24 +106,87 @@ source("r/shf_thy_nod.R")
+
+### Variables
+Variable evaluated was based on findings from a systematic review evaluating predictors of thyroid cancer in patients
+with thyroid nodules. Data on the following variables were collected: patient demographics (age, gender, ethnicity),
+nodule presentation (incidental nodule, palpable nodule, rapid enlargement, compressive symptoms, vocal paresis), past
+medical history (hypertension, Graves' disease, Hashimotos' thyroiditis, family history of thyroid cancer, exposure to
+neck radiation), biochemistry (thyroid stimulating hormone, lymphocytes, monocytes), ultrasound characteristics (British
+Thyroid Association ultrasound (BTA U), nodule size, solitary nodule, nodule consistency, cervical lymphadenopathy),
+Royal College of Pathology (RCP) FNAC  classification, type of thyroid surgery, and histological diagnosis.
+
+### Data source
+
+Data was collected from patients' case notes and electronic patients' database. Data was collected using a standardised
+data collection proforma. This was initially piloted on 30 patients  and revised to improve data entry. In addition a
+number of variables that were not standard collected during workout of patients were not further checked; these include
+body mass index (BMI), serum thyroglobulin, serum triiodothyronine (T3), thyroxine (T4), thyroglobulin antibody (TgAb),
+thyroid peroxidase antibody (TP0Ab), and urinary iodine. 
+
+### Study size
+We sought to have a large data set of at least 100 thyroid nodules with a cancer diagnosis using consecutive sampling
+technique. 
+We aimed for a total of 1500 patients with thyroid nodules to achieve our target sample size. With the use
+of modern statistical techniques, we proposed such number will be appropriate to detect important variables if it
+exists.
+
+### Data analysis
 
 Data was cleaned and analysed using the R Statistical Software @r_citation and the Tidyverse (@tidyverse),  Tidymodels
 (@tidymodels) collection of packages.
 
 ### Modelling
 
-Description of the different models and how they are assessed can go here.
-
-A selection of modern statistical inference approaches have been selected and tested for this work. To fit the models
-data is split into training and testing cohorts in a ratio of `r train`:`r test` and each model is fitted using the
-training cohort. The predictive accuracy of the fitted model is then assessed in the test cohort. **TODO** Expand on this.
+We used a selection of statistic modelling techniques to evaluate association between variables and thyroid cancer in patients with thyroid nodules.
+We used the training and test methodology to split the patient population into training and testing cohorts in a ratio
+of `r train`:`r test` and each model is fitted using the
+training cohort. This split ratio is generally used in traditional machine learning techniques. The training set of the
```

Comment:
Ratio's vary on preference `70:30` for some others `80:20` no hard and fast rules for these, the key thing is that you can't use your whole data set to develop your model and assess its accuracy.